changelogs/fragments/423-postgresql_pg_hba_comment_parsing.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+- postgresql_pg_hba - comment parsing assumed any hash starts a comment, but could be in quoted values (https://github.com/ansible-collections/community.postgresql/issues/420)
```

```suggestion
- postgresql_pg_hba - comment parsing assumed any hash starts a comment, but could be in quoted values (https://github.com/ansible-collections/community.postgresql/issues/420).
```
To make all changelog entries consitent.

Let's wait for @betanummeric until tomorrow, then will merge the PR

---

The changelog should focus on what changed, this is more of an issue description. Suggestion:
> fix pg_hba.conf parsing: support double-quoted values, including escaped quotes

---

+1 @stelb what do you think ^ ?

---

Well, ok. I took a look in one changelog and it seemed to describe the fixed problem too.  I'm fine with your suggestion :)

- All others prefix it by module name
- a colon needs to be in a quoted string in yaml.

```
bugfixes:
  - "postgresql_pg_hba - fix pg_hba.conf parsing: support double-quoted values, including escaped quotes"
```